diff --git a/zipl/boot/Makefile b/zipl/boot/Makefile
index 6474aad..96edc59 100644
--- a/zipl/boot/Makefile
+++ b/zipl/boot/Makefile
@@ -60,7 +60,7 @@ stage3.exec: head.o stage3.o kdump3.o libc.o sclp.o sclp_stage3.o \
 		2) SFLAGS="-nostdlib -Wl,-T,stage2.lds";; \
 		3) SFLAGS="-nostdlib -Wl,-T,stage3.lds";; \
 	esac; \
-	$(LINK) $$SFLAGS -m64 $^ -o $@
+	$(LINK) -no-pie $$SFLAGS -m64 $^ -o $@
 
 %.bin:	%.exec
 	$(OBJCOPY) -O binary \
